### PR TITLE
Various bug fixes

### PR DESCRIPTION
**Pull request by miparnisari**
- Added "region" as input beside the API key
- Fixed the URL endpoint (Fixes https://github.com/Microsoft/Cognitive-Vision-Python/issues/2)
- Fixed the error displayed when an error occurs (Fixes https://github.com/Microsoft/Cognitive-Vision-Python/issues/9)

### CODE REVIEW

File: Jupyter Notebook/Computer Vision API Example.ipynb [hunk 1]
```diff
@@ -51,8 +51,8 @@
    "outputs": [],
    "source": [
     "# Variables\n",
-    "\n",
-    "_url = 'https://westcentralus.api.cognitive.microsoft.com/vision/v1/analyses'\n",
+    "_region = 'westus' #Here you enter the region of your subscription\n",
```
Comment (cthrash) [inline Jupyter Notebook/Computer Vision API Example.ipynb]: Is there a good reason for this not being `westcentralus`?

Comment (miparnisari) [reply Jupyter Notebook/Computer Vision API Example.ipynb]: Um in general it seems to me that the default value for the region (or at least the most used) is westus. But I can change it back if you want to.

Comment (cthrash) [reply Jupyter Notebook/Computer Vision API Example.ipynb]: Please do.  This was an intentional [change](https://github.com/Microsoft/Cognitive-Vision-Windows/commit/1da8394bff5441ec07cfcb32cc8ed4fe75efef29).

Comment (miparnisari) [reply Jupyter Notebook/Computer Vision API Example.ipynb]: Done!

**Comment by cthrash** [comment]
Thanks for your contribution!

**Pull request merged**